lib/chess/store/game.ex

```diff
@@ -12,6 +12,7 @@ defmodule Chess.Store.Game do
 
   schema "games" do
     field :board, :map
+    field :turn, :string
```

Comment:
What do you think about having a default here too?

You have a default in the database already. The difference is that with the schema default, calling an empty `%Game{}` will pre-fill it with `turn: white`; 
without the schema default, `%Game{}` will have `turn: nil` until it's persisted to the database, and I believe _then_ it'll have `turn: white` returned. This behavior is a little surprising sometimes, so I tend to have the default in both places because I expect the empty struct to have my defaults.

Reply:
Sounds good! I changed this and also used the same technique to set the default board. This makes my changeset code much simpler!